webhook/resources/volume/validator.go

```diff
@@ -264,6 +268,16 @@ func (v *volumeValidator) Update(request *admission.Request, oldObj runtime.Obje
 		}
 	}
 
+	// Allow backup block size mutation only when the existing obj is not set, or correcting the existed invalid value
+	isValidOldBackupBlockSize := types.ValidateBackupBlockSize(oldVolume.Spec.Size, oldVolume.Spec.BackupBlockSize) == nil
+	if isValidOldBackupBlockSize && oldVolume.Spec.BackupBlockSize != newVolume.Spec.BackupBlockSize {
+		err := fmt.Errorf("changing backup block size for volume %v is not supported", oldVolume.Name)
+		return werror.NewInvalidError(err.Error(), "")
```

Comment:
I am thinking how Longhorn can handle the case containing the following conditions:
1. The default block size is set to 16Mi
2. The backup CR contains an empty `Spec.BackupBlockSize`, and the actual size it uses is 2Mi
3. There is no corresponding volume in the cluster, or `volume.Spec.BackupBlockSize` is not 2Mi